packages/blade/src/components/Typography/Heading/_KitchenSink.Heading.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import { composeStories } from '@storybook/react';
+import * as headingStories from './Heading.stories';
+import { Box } from '~components/Box';
+import { Heading } from '~components/Typography';
+
+const allStories = Object.values(composeStories(headingStories));
+
+export const Heading = (): JSX.Element => {
```

Comment:
It's conflicting with the imported `Heading` component.

```suggestion
export const HeadingStory = (): JSX.Element => {
```